srcpkgs/jetbrains-jdk-bin/template

```diff
@@ -35,5 +35,12 @@ do_install() {
 	vmkdir etc/profile.d
 	cat > ${DESTDIR}/etc/profile.d/10_jbrsdk.sh <<EOF
 export IDEA_JDK=\${IDEA_JDK=/usr/lib/jvm/jbrsdk}
+export PHPSTORM_JDK=\${IDEA_JDK}
+export WEBIDE_JDK=\${IDEA_JDK}
+export PYCHARM_JDK=\${IDEA_JDK}
+export RUBYMINE_JDK=\${IDEA_JDK}
+export CLION_JDK=\${IDEA_JDK}
+export DATAGRIP_JDK=\${IDEA_JDK}
+export GOLAND_JDK=\${IDEA_JDK}
```

Comment:
Doesn't it make more sense for each application to
- either ship its own profile file (making use of `$IDEA_JDK`)
- or use a start script that prefxes the command with `XXX_JDK=$IDEA_JDK`
?

Reply:
Agree, change from symlink to variable was supposed to make IDEs less dependent on jdk changes, but this way they are more dependent.